### DISCUSSION TITLE
Glass plate?

### DISCUSSION THREAD

**Discussion by dev-zetta** [dz0ny/klipper-prusa-mk3s#18] [Q&A]
Hey folks,

so I'd like to use glass plate to print on, how do I do that? Is PINDA probe able to detect the steel sheet below the glass reliably?

If not, can I disable PINDA and use some other probe, like TAP, or klicky?

I was thinking about glueing the steel sheet to the glass, since I'll be printing onto glass only.

I'm new to klipper so excuse me for placing this dumb question.

**Comment by dz0ny** [comment]
You can try moving the head to one of the magnets on the plate with the glass plate, if the LED lights up (on pinda) you should be fine. You will probably need to tune sense and definitely Z-offset.

**Reply by dev-zetta** [reply]
Thank you :) I will actually glue that steel sheet to the glass sheet, so the steel sheet will stay, however I'm worried about the nozzle hitting the glass. Is there any distance limit for PINDA to detect the steel sheet? I remember seeing 3 mm threshold defined in prusa's firmware.

Now when I probe the bed with glass sheet on it, the nozzle always hit the glass, no matter what z_offset I set - with steel sheet, I have value around 1.5, but the glass adds another 3 mm to it, so even when I set the z offset to zero, I hit the glass.

**Reply by dz0ny** [reply]
The inductive probe could be 3mm yes, as one usually buys one based on the distance you want to measure your object. For example 5mm one, but huge in prusa standards :D https://heschen.com/products/ljc18a3-5-z-cy

**Reply by dev-zetta** [reply]
Thanks for the tip! :) Btw, I saw that someone has remixed prusa's extruder part that holds PINDA probe so it can mount BLTouch. I was wondering, would it be hard to switch to BLTouch or Klicky probe?

https://www.printables.com/cs/model/51302-bltouch-mount-prusa-i3-mk3s-extruder-r6-remix

**Reply by dz0ny** [reply]
Pretty much follow the https://www.klipper3d.org/BLTouch.html, there are a bunch of exposed pins(for mmu2 for example) on the rambo board so you should be fine.
